### PR TITLE
logInform corrected to login.ejs event listener

### PR DESCRIPTION
**Pull request by richardgabaree**

### CODE REVIEW

File: config/config.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "development": {
     "username": "root",
-    "password": "",
+    "password": "J@hntanious1",
```
Comment (trevorsmithbanjo) [inline config/config.json]: This password should be removed.

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "p2_org_decider",
   "version": "1.0.0",
-  "description": "",
+  "description": "J@hntanious1",
```
Comment (trevorsmithbanjo) [inline package.json]: It should be removed here as well.

**Review by trevorsmithbanjo — CHANGES_REQUESTED** [review]
Your password needs to be removed from the config.json and package.json.

**Comment by richardgabaree** [comment]
I deleted and did another push

**Review by trevorsmithbanjo — APPROVED** [review]
👍🏻

**Pull request merged**